schema/project.js

```diff
@@ -45,6 +45,7 @@ class Project extends BaseModel {
         issueDate: { type: ['string', 'null'], format: 'date-time' },
         expiryDate: { type: ['string', 'null'], format: 'date-time' },
         revocationDate: { type: ['string', 'null'], format: 'date-time' },
+        raDate: { type: ['string', 'null'], pattern: date.yearMonthDay },
```

Comment:
Oh. I've fucked it here haven't I?

There's no good reason for this to be inconsistent.

Reply:
lol

Reply:
we use YYYY-MM-DD for other dates like DOB etc